OpenRA.Mods.Common/SpriteLoaders/GifLoader.cs

```diff
@@ -0,0 +1,308 @@
+﻿using OpenRA.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenRA.Mods.Common.SpriteLoaders
+{
+	class GifLoader : ISpriteLoader
```

Comment:
This should probably be public in convention with the other loaders so other assemblies can use this.